app/lib/services.rb

```diff
@@ -1,9 +1,9 @@
 require "gds_api/support"
-require "gds_api/publishing_api_v2"
+require "gds_api/publishing_api"
 
 module Services
   def self.publishing_api_v2
-    @publishing_api_v2 ||= GdsApi::PublishingApiV2.new(
+    @publishing_api_v2 ||= GdsApi::PublishingApi.new(
```

Comment:
```ruby
def self.publishing_api_v2
  GdsApi.publishing_api_v2
end
```

[Much shorter!](https://github.com/alphagov/gds-api-adapters/blob/25520c648e673960832494b5674c303301073363/lib/gds_api.rb#L148)

If you feel like it, I would even go further and update where we do `Services.publishing_api_v2` in the code and use `GdsApi.publishing_api_v2` but that might be a lot of work.

Reply:
I did this for travel-advice-publisher here https://github.com/alphagov/travel-advice-publisher/pull/602.

Reply:
It wasn't too much work.